app_service/app_handler.go

```diff
@@ -0,0 +1,13 @@
+package app_service
+
+import "github.com/gin-gonic/gin"
+
+type AppHandler interface {
```

Comment:
What is the point of this interface? And if you see a point - then the interface defines a Library and not a general appHandler